felix/fv/wireguard_test.go

```diff
@@ -1748,6 +1778,7 @@ func wireguardTopologyOptions(routeSource string, ipipEnabled, wireguardIPv4Enab
 	// Indicate wireguard is enabled
 	topologyOptions.WireguardEnabled = wireguardIPv4Enabled
 	topologyOptions.WireguardEnabledV6 = wireguardIPv6Enabled
+	topologyOptions.wireguardThreadingEnabled = wireguardThreadingEnabled
```

Comment:
```suggestion
	topologyOptions.WireguardThreadingEnabled = wireguardThreadingEnabled
```

Reply:
Thanks, I need to fix my local code editor. It's not giving intellisense for calico tests.